misc/python/materialize/zippy/scenarios.py

```diff
@@ -83,8 +90,12 @@ def bootstrap(self) -> list[ActionOrFactory]:
 
     def config(self) -> dict[ActionOrFactory, float]:
         return {
-            MzStart: 1,
-            MzStop: 15,
+            # 50 so that Mz is started quickly after being killed
```

Comment:
What does it mean "is started quickly"? That the likelihood is high that an `MzStart` action takes place?
The numbers are counts / likelihoods of the actions?
We could also incorporate that info in the message name if `config` is returning nothing else (`get(_configured)_actions_with_...`).

Reply:
Here is what was going on here:
1. Previously, if Mz was killed, most of the other actions became impossible so MzStart was picked quickly after an mz kill.
2. I attempted to make it so that Kafka topic creation and `kafka-ingest` also does not require that Mz is running. This caused a situation where, once Mz was killed, topic creation and ingestion actions were picked repeatedly and with high probability and Mz was not restarted quickly enough.

However, it turns out that now testdrive's kafka operations require a running Mz instance (or so it seems), so I reverted the change and now the balance is as before ,so MzStart does not require that probability boost.